zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/web/rest/DefaultZaakTypeLinkResource.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Dimpact.
```

Comment:
Did you just copy all these files, or have they been changed as well? I don't know if we need to keep the 'Dimpact' header when we changed things, but we'd need to check with management.

Reply:
That's fair. I did make some changes to some of the files.

Reply:
It's ok to leave these here.